extensions/job_management/standard/sections/clause_12_security_considerations.adoc

```diff
@@ -0,0 +1,119 @@
+== Security Considerations
+
+See <<OAProc-1,OGC API - Processes - Part 1: Core>>, Clause 10.4.
+
+Since creating and updating jobs will change the jobs available to a client, servers will - in almost all cases - restrict the access to these operations.
+
+Users making modifications to process resources need to:
+
+. Be authenticated,
+. Have "modification privileges" on the jobs offered through the API,
+. Have access to one or more of the POST and/or PUT methods on the jobs /jobs/{jobId} endpoints.
+
+The API definition, as defined in Clause 7.3 from <<OAProc-1>>, must reflect this in the resource paths and their available methods.
+
+Examples in the Clauses specifying the requirements classes focus on the mechanics of the POST, PUT, and DELETE methods and exclude authentication. Since authentication will typically be required for all DRU requests, this section provides some examples/guidance:
+
+The OpenAPI definition exposed by the serve will declare the authentication schemes that an implementation of the Processes - Part 4 (JM) supports for each operation (or for all operations in the API implementation).
```

Comment:
Only if it actually offers an OpenAPI definition ;-)

Reply:
I propose we remove following examples entirely. It is not relevant to go into details about security representations here, since there are many implementations out there, and it creates false expectations when reading this document.

There are ongoing discussions in OGC API - Common/Security SWG about similar concerns, and that OpenAPI should not be considered the de-facto place to look for authentication schemes (it can provide them for convenience, but it is not the "reference").